exoscale/resource_exoscale_sks_cluster_test.go

```diff
@@ -49,6 +49,8 @@ resource "exoscale_sks_cluster" "test" {
     test = "%s"
   }
 
+  enable_kube_proxy = true
```

Comment:
can we have a test with it true and one with false ?

Reply:
We can but it's not a meaningful test in that context: this attribute is only significant during cluster creation, and the result of it being set (or unset) is not checkable through the API.

So for now this will (temporarily) have to do.